internal/overlord/servstate/manager.go

```diff
@@ -34,6 +34,12 @@ type ServiceManager struct {
 
 	randLock sync.Mutex
 	rand     *rand.Rand
+
+	logMgr LogManager
+}
+
+type LogManager interface {
+	ServiceStarted(serviceName string, buffer *servicelog.RingBuffer)
```

Comment:
We call the log ring buffer as `logs` everywhere in the existing code.  I don't mind which name we pick too much so please feel free to suggest something else, but I would prefer to pick one single name for this value across the implementation, instead of changing depending on context (`buffer` is probably not it either, though).

Reply:
Ok, I changed the argument name to `logs` everywhere inside the `servstate` package. I think inside the `logstate` package, `buffer` makes more sense as `logs` could refer to anything. I interpreted "across the implementation" to mean just the `servstate` package, as the buffer is called `rb` elsewhere.